config/maintainers.json

```diff
@@ -63,6 +63,18 @@
         "python/test-runner",
         "python/research-exercise"
       ]
+    },
+    {
+      "github_username": "pvcarrera",
+      "name": "Pablo Vicente",
+      "link_text": "",
+      "link_url": "",
+      "avatar_url": "https://s.gravatar.com/avatar/5b36c6cfb4083458ae9e12a377692efc?s=400",
+      "bio": "I am a software developer with experience on web development and API design. When I'm not coding you can find me in the mountains.",
+      "contributions": [
+        "ruby/test-runner",
```

Comment:
Just a small check. I don't see your name in the Ruby test runner contributors: https://github.com/exercism/ruby-test-runner/graphs/contributors. Is that correct?

Reply:
Sorry I wasn't sure, at first I didn't add this line but then I saw all the contributors with that line and I thought that it was part of the exercise.

Reply:
No worries!